.github/workflows/set-start-date.yml

```diff
@@ -0,0 +1,84 @@
+name: Set the Start Date in the project
+
+on:
+  issues:
+    types: [assigned]
+jobs:
+  set_date:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Get project data
+        env:
+          GITHUB_TOKEN: ${{ secrets.KEPTN_BOT_PROJECT_TOKEN }}
+          ORGANIZATION: keptn
+          PROJECT_NUMBER: 18
+        run: |
+          gh api graphql -f query='
+            query($org: String!, $number: Int!) {
+              organization(login: $org){
+                projectV2(number: $number) {
+                  id
+                  fields(first:20) {
+                    nodes {
+                      ... on ProjectV2Field {
+                        id
+                        name
+                      }
+                      ... on ProjectV2SingleSelectField {
+                        id
+                        name
+                        options {
+                          id
+                          name
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }' -f org=$ORGANIZATION -F number=$PROJECT_NUMBER > project_data.json
+          echo 'PROJECT_ID='$(jq '.data.organization.projectV2.id' project_data.json) >> $GITHUB_ENV
+          echo 'DATE_FIELD_ID='$(jq '.data.organization.projectV2.fields.nodes[] | select(.name== "Start Date") | .id' project_data.json) >> $GITHUB_ENV
+      
+      - name: Get date
+        run: echo "DATE=$(date +"%Y-%m-%d")" >> $GITHUB_ENV
+        
+      - name: Add item id
```

Comment:
Here, we are extracting the project item ID of the issue and then saving it into an env variable named `ITEM_ID`